### PR TITLE
Cut(eos_designs): Remove the deprecated ipv6_address_virtual key from SVI settings

### PR DESCRIPTION
**Pull request by MaheshGSLAB**
## Change Summary

Remove deprecated key(ipv6_address_virtual) for svi-settings schema.

## Related Issue(s)

Fixes #<ISSUE ID>

## Component(s) name

`arista.avd.eos_designs

## Proposed changes
Remove deprecated key(ipv6_address_virtual) for svi-settings schema.
Removed tests from deprecated vars

## How to test
Run deprecated vars molecule

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by ClausHolbechArista — APPROVED** [review]

**Review by Vibhu-gslab — APPROVED** [review]
LGTM!

File: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ The following data model keys have been removed from `eos_designs` in v5.0.0.
 | old key 2(defs_adapter_config) | new key(TODO) |
 | old key 3.1(defs_node_type) | new key(TODO) |
 | old key 3.2(defs_node_type) | new key(TODO) |
-| old key 4(defs_svi_settings) | new key(TODO) |
+| svi_settings.ipv6_address_virtual | svi_settings.ipv6_address_virtuals |
```
Comment (Shivani-gslab) [inline ansible_collections/arista/avd/docs/porting-guides/5.x.x.md]: please mention the data-model names here

**Review by Shivani-gslab — CHANGES_REQUESTED** [review]

File: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md (hunk 1 above)
Comment (Shivani-gslab) [reply ansible_collections/arista/avd/docs/porting-guides/5.x.x.md]: same as this - https://github.com/aristanetworks/avd/pull/4319#discussion_r1703981795

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/docs/porting-guides/5.x.x.md]:
```suggestion
| <network_services_key>[].vrfs[].svis[].ipv6_address_virtual | <network_services_key>[].vrfs[].svis[].ipv6_address_virtuals |
| svi_profiles[].ipv6_address_virtual | svi_profiles[].ipv6_address_virtuals |

```

File: python-avd/pyavd/_eos_designs/structured_config/network_services/vlan_interfaces.py [hunk 2]
```diff
@@ -135,10 +135,6 @@ def _check_virtual_router_mac_address(vlan_interface_config: dict, variables: li
 
         # Only set Anycast v6 GW if VARPv6 is not set
         if vlan_interface_config.get("ip_virtual_router_addresses") is None:
-            if (ipv6_address_virtual := svi.get("ipv6_address_virtual")) is not None:
-                # The singular ipv6_address_virtual is deprecated from eos_cli_config_gen. So set as list item into the new key.
-                vlan_interface_config["ipv6_address_virtuals"] = [ipv6_address_virtual]
-
             if (ipv6_address_virtuals := svi.get("ipv6_address_virtuals")) is not None:
                 vlan_interface_config.setdefault("ipv6_address_virtuals", []).extend(ipv6_address_virtuals)
```
Comment (laxmikantchintakindi) [inline python-avd/pyavd/_eos_designs/structured_config/network_services/vlan_interfaces.py]:
```suggestion
                vlan_interface_config["ipv6_address_virtuals"] = ipv6_address_virtuals
```

**Review by Shivani-gslab — APPROVED** [review]
LGTM!

File: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md [hunk 3]
```diff
@@ -56,7 +56,9 @@ The following data model keys have been removed from `eos_designs` in v5.0.0.
 | old key 2(defs_adapter_config) | new key(TODO) |
 | old key 3.1(defs_node_type) | new key(TODO) |
 | old key 3.2(defs_node_type) | new key(TODO) |
-| old key 4(defs_svi_settings) | new key(TODO) |
+| <network_services_key>[].vrfs[].svis[].ipv6_address_virtual | <network_services_key>[].vrfs[].svis[].ipv6_address_virtuals |
+| svi_profiles[].ipv6_address_virtual | svi_profiles[].ipv6_address_virtuals |
+
```
Comment (laxmikantchintakindi) [inline ansible_collections/arista/avd/docs/porting-guides/5.x.x.md]:
```suggestion
```

**Review by ClausHolbechArista — APPROVED** [review]

**Review by laxmikantchintakindi — APPROVED** [review]

**Review by ClausHolbechArista — APPROVED** [review]

**Pull request merged**